openmeter/ent/schema/billing.go

```diff
@@ -292,13 +292,39 @@ func (m TotalsMixin) Fields() []ent.Field {
 	}
 }
 
+type InvoiceLineBaseMixin struct {
+	mixin.Schema
+}
+
+func (InvoiceLineBaseMixin) Fields() []ent.Field {
+	return []ent.Field{
+		field.Time("period_start"),
```

Comment:
should we rename it to BillingPeriod/InvoicePeriod or ServicePeriod depending which one this one is?

Reply:
It's service period, changed. However only for the split line group as the other line types would require additional changes.